dask/dataframe/core.py

```diff
@@ -876,13 +876,20 @@ def head(self, n=5, npartitions=1, compute=True):
         compute : bool, optional
             Whether to compute the result, default is True.
         """
+        return self._head(n=n, npartitions=npartitions, compute=compute, safe=True)
+
+    def _head(self, n, npartitions, compute, safe):
```

Comment:
This is a bit ugly, but `DataFrame.describe()` calls `head`. If we have too few values, we previously got a warning. To avoid this, we need to pass whether or not to warn all the way down to the `head` called on each chunk (`safe_head` and `_unsafe_head`).

We don't want `warn` as part of the public API, so we need `DataFrame._head`.